stdlib/2/subprocess.pyi

```diff
@@ -57,10 +57,13 @@ STDOUT = ...  # type: int
 
 class CalledProcessError(Exception):
     returncode = 0
-    cmd = ...  # type: str
-    output = ...  # type: str  # May be None
+    cmd = ...  # type: Any
```

Comment:
OK, I'll add them back! I thought they might be redundant with the actual types in the constructor.